controllers/meetup.js

```diff
@@ -0,0 +1,97 @@
+const ApiError = require('../error/apiError')
+const {meetupBodyDTO, meetupQueryDTO, meetupParamDTO} = require('../dto/meetup.dto')
+const MeetupService = require('../service/meetup')
+const jwt = require('jsonwebtoken')
+
+
+const decodedUser = (token) => {
+    return jwt.verify(token.split(' ')[1], process.env.SECRET_KEY);
```

Comment:
```suggestion
    const [,tokenRaw] = token.split(' ');
    return jwt.verify(tokenRaw, process.env.SECRET_KEY);
```

читабельнее и понятнее